micro-rdk-installer/src/main.rs

```diff
@@ -47,7 +49,19 @@ struct WriteCredentials {
 }
 
 #[derive(Args)]
-struct WriteFlash {}
+struct WriteFlash {
+    #[arg(long = "app-config")]
+    config: String,
+    #[arg(long = "bin")]
+    binary_path: String,
+    #[arg(long = "nvs-size", default_value = "32768")]
```

Comment:
I'm about to learn something else about Rust! What does `#[arg(...)]` do? If `long` specifies a different name for the thing, why are the names different? Why is the "long" version of `binary_path` just "bin"?

Reply:
This is a feature of clap, the crate that I'm using for building the installer CLI. `arg` is essentially sugar for auto-parsing a CLI argument into a corresponding field on the struct. As for the names, I can make them more consistent (e.g. "bin" should probably be the short name and the long name should be "binary-path")